public_src/components/editor/editor.component.ts

```diff
@@ -161,4 +170,14 @@ export class EditorComponent implements OnInit, AfterViewInit {
       }, 250);
     }
   }
+
+  private routeCreationWizard(): any {
+    this.modalRefRouteWiz = this.modalService.show(RouteWizardComponent, { class: 'modal-lg', ignoreBackdropClick: true });
+    this.appActions.actSetWizardMode('route wizard');
+  }
+
+  private routeMasterCreationWizard(): any {
```

Comment:
`routeCreationWizard` + `routeMasterCreationWizard` should be `public` ... use private only when the fn is used only in that one file and nowhere else (not in HTML code for the TS file)... some explanation maybe here - https://stackoverflow.com/a/42890630/4114514